doc/changelog.md

```diff
@@ -145,6 +145,9 @@
 * Use Tablegen to define MLIR passes instead of C++ to reduce overhead of adding new passes.
   [#157](https://github.com/PennyLaneAI/catalyst/pull/157)
 
+* Add support for JAX v4.10

```

Comment:
"Adding support" for JAX `v0.4.10` is not the right phrasing, since we are switching our (hard) dependency chain to this version rather than adding support for it.

Reply:
Thanks! I see what you mean, what do you think about the new comment here:

> Update JAX to `v.0.4.10`.

https://github.com/PennyLaneAI/catalyst/pull/143/commits/3b9ab5e7c132214034304272bcc0f8ec9a149a5a

Reply:
Thanks!